cli/cmd_install.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/benchkram/bob/bob"
+	"github.com/benchkram/bob/pkg/boblog"
+	"github.com/spf13/cobra"
+)
+
+var installCmd = &cobra.Command{
+	Use:   "install",
+	Short: "Install all dependencies",
+	//Args:  cobra.ExactArgs(1),
+	Long: ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		runInstall()
+	},
+}
+
+func runInstall() {
+	b, err := bob.Bob()
+	if err != nil {
+		boblog.Log.Error(err, "Unable to initialise bob")
```

Comment:
Handle exit code correctly in case of an error.. check https://github.com/benchkram/bob/blob/affb1e2ff3650c6e90ad1ca35e80ed16556c1448/cli/cmd_build.go#L63